BitmovinConvivaAnalytics.podspec

```diff
@@ -12,16 +12,16 @@ Conviva Analytics Integration for the Bitmovin Player iOS SDK
   s.author           = { 'Bitmovin' => 'david.steinacher@bitmovin.com' }
   s.source           = { git: 'https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva.git', tag: s.version.to_s }
 
-  s.ios.deployment_target = '9.0'
-  s.tvos.deployment_target = '9.0'
+  s.ios.deployment_target = '10.0'
+  s.tvos.deployment_target = '10.0'
 
   s.source_files = 'BitmovinConvivaAnalytics/Classes/**/*.swift'
   s.resources = 'BitmovinConvivaAnalytics/Assets/*'
   s.swift_version = '4.2'
   s.cocoapods_version = '>= 1.4.0'
 
-  s.ios.dependency 'BitmovinPlayer', '~> 2.51'
-  s.tvos.dependency 'BitmovinPlayer', '~> 2.51'
+  s.ios.dependency 'BitmovinPlayer', '~> 2.57.1'
+  s.tvos.dependency 'BitmovinPlayer', '~> 2.57.1'
```

Comment:
Please update the version in the `README` accordingly as well.

Reply:
Which line needs to be updated in the `README`?
This PR would not have a version bump so it looks like updating the `Compatibility` section wouldn't be needed until there is a release.

Reply:
I was thinking of something like: (_Even though the links do not exist yet - and assuming `1.3` as the next version_)

---

### Compatibility
This version of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 2.57.1`. If you need support for prior `2.57.1` please use [v1.2.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.2.0). For `BitmovinPlayer` version `>= 2.57.1` please use [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.

Reply:
I understand. With assuming 1.3 is the next version I updated the text like so:

The most recent version of the Conviva Analytics Integration depends on `BitmovinPlayer` version `>= 2.57.1`.
For `BitmovinPlayer` version `>= 2.57.1` please use [v1.3.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.3.0) or later.

If you need support for prior to `2.57.1` please use [v1.2.0](https://github.com/bitmovin/bitmovin-player-ios-analytics-conviva/tree/1.2.0).